audit_log/tests/runtests.py

```diff
@@ -8,68 +8,75 @@
 
 
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))
-sys.path.append(os.path.abspath(os.path.join(BASE_DIR, '..')))
+sys.path.append(os.path.abspath(os.path.join(BASE_DIR, "..")))
 
 # Unfortunately, apps can not be installed via ``modify_settings``
 # decorator, because it would miss the database setup.
-CUSTOM_INSTALLED_APPS = (
-    'audit_log',
-)
+CUSTOM_INSTALLED_APPS = ["audit_log"]
 
-ALWAYS_INSTALLED_APPS = (
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-)
+ALWAYS_INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+]
 
-ALWAYS_MIDDLEWARE_CLASSES = (
-    'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
-    'audit_log.middleware.UserLoggingMiddleware',
-)
+ALWAYS_MIDDLEWARE_CLASSES = [
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "audit_log.middleware.UserLoggingMiddleware",
+]
```

Comment:
Propose to not introduce any code-style refactoring in this PR (i.e. keep single quotes)